shared/chat/conversation/messages/wrapper/exploding-meta/index.js

```diff
@@ -46,9 +28,7 @@ type State = {
 }
 
 class ExplodingMeta extends React.Component<Props, State> {
```

Comment:
testing on iOS, the left side of this is cut off 
<img width="459" alt="image" src="https://user-images.githubusercontent.com/11968340/49451331-f3b59780-f7ac-11e8-9bc9-622599359c05.png">